state/indexer/block/kv/kv_test.go

```diff
@@ -229,26 +229,14 @@ func TestBlockIndexerMulti(t *testing.T) {
 			q:       query.MustCompile("block.height = 1"),
 			results: []int64{1},
 		},
-		"query return all events from a height - exact - no match.events": {
-			q:       query.MustCompile("block.height = 1"),
-			results: []int64{1},
-		},
```

Comment:
I don't understand why these test cases are no longer needed

Reply:
BEcause they are the exact duplicate of the existing tests - they were there because we used to test with and without the match-events keywords and then when I fixed the tests for not use the keyword, I did not delete the duplicate ones.